physical/postgresql/postgresql.go

```diff
@@ -174,6 +186,7 @@ func NewPostgreSQLBackend(conf map[string]string, logger log.Logger) (physical.B
 		list_query: "SELECT key FROM " + quoted_table + " WHERE path = $1" +
 			" UNION ALL SELECT DISTINCT substring(substr(path, length($1)+1) from '^.*?/') FROM " + quoted_table +
 			" WHERE parent_path LIKE $1 || '%'",
+		haClient: db,
```

Comment:
Did this mean to be `db`? It seems like `haDB` is unused and is intended to be chosen as the client here, but as-is this doesn't actually change anything except instantiate a new client.